--- a/sql/CMakeLists.txt	2011-12-16 18:16:49.000000000 -0800
+++ b/sql/CMakeLists.txt	2011-12-16 18:32:51.000000000 -0800
@@ -265,7 +265,7 @@
 # Avoid completely empty directories and install dummy file instead.
 SET(DUMMY_FILE ${CMAKE_CURRENT_BINARY_DIR}/.empty ) 
 FILE(WRITE ${DUMMY_FILE} "")
-INSTALL(FILES ${DUMMY_FILE} DESTINATION data/test COMPONENT DataFiles)
+INSTALL(FILES ${DUMMY_FILE} DESTINATION ${INSTALL_MYSQLDATADIR} COMPONENT DataFiles)
 
 # Install initial database on windows
 IF(NOT CMAKE_CROSSCOMPILING)
@@ -298,7 +298,7 @@
   COMPONENT DataFiles PATTERN "initdb.dep" EXCLUDE  PATTERN "bootstrap.sql" EXCLUDE)
 ELSE()
   # Not windows or cross compiling, just install an empty directory
-  INSTALL(FILES ${DUMMY_FILE} DESTINATION data/mysql COMPONENT DataFiles)
+  INSTALL(FILES ${DUMMY_FILE} DESTINATION ${INSTALL_MYSQLDATADIR} COMPONENT DataFiles)
 ENDIF()
 ENDIF()
 
