--- a/Makefile	2011-08-10 22:30:09.000000000 -0700
+++ b/Makefile	2011-08-10 22:40:02.000000000 -0700
@@ -82,8 +82,29 @@
     CFLAGS += -DSTATEFILE=\"$(STATEFILE)\"
 endif
 
+ifneq ($(DEFAULT_MAIL_COMMAND),)
+    CFLAGS += -DDEFAULT_MAIL_COMMAND=\"$(DEFAULT_MAIL_COMMAND)\"
+endif
+
+ifneq ($(COMPRESS_COMMAND),)
+    CFLAGS += -DCOMPRESS_COMMAND=\"$(COMPRESS_COMMAND)\"
+endif
+
+ifneq ($(COMPRESS_EXT),)
+    CFLAGS += -DCOMPRESS_EXT=\"$(COMPRESS_EXT)\"
+endif
+
+ifneq ($(UNCOMPRESS_COMMAND),)
+    CFLAGS += -DUNCOMPRESS_COMMAND=\"$(UNCOMPRESS_COMMAND)\"
+endif
+
+ifeq ($(OS_NAME),Darwin)
+    INSTALL = install
+    LDFLAGS += -undefined dynamic_lookup
+endif
+
 BINDIR = $(BASEDIR)/sbin
-MANDIR = $(BASEDIR)/man
+MANDIR = $(BASEDIR)/share/man
 
 #--------------------------------------------------------------------------
 
@@ -92,7 +113,7 @@
 
 ifeq ($(RPM_OPT_FLAGS),)
 CFLAGS += -g
-LDFLAGS = -g
+LDFLAGS += -g
 endif
 
 ifeq (.depend,$(wildcard .depend))
